client/platform/web-girder/views/DataDetails.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <v-card class="data-details">
+    <v-toolbar
+      flat="flat"
+      dark="dark"
+      dense="dense"
+      color="primary"
+    >
+      <v-toolbar-title class="subtitle-1">
+        <v-icon class="pr-2 mdi-18px">
+          {{ icon }}
+        </v-icon>{{ title }}
+      </v-toolbar-title>
+    </v-toolbar>
+    <girder-markdown
+      v-if="details && details.description"
+      :text="details.description"
+      class="mx-3 mt-2"
+    />
+    <girder-detail-list
+      :rows="info"
+      title="Info"
+    />
+    <slot name="actions" />
+  </v-card>
+</template>
+
+<script>
+import Vue from 'vue';
+import {
+  GirderDetailList,
+  GirderMarkdown,
+  mixins,
+} from '@girder/components/src';
+
+const {
+  dateFormatter,
+  sizeFormatter,
+  usernameFormatter,
+} = mixins;
+
+/**
+ * @type {Array<{
+ *  value: String,
+ *  name: String,
+ *  transform: Function
+ * }>}
+ */
+export const DefaultInfoKeys = [
+  {
+    value: 'size',
+    name: 'Size: ',
+    transform: sizeFormatter.methods.formatSize,
+  },
+  {
+    value: 'created',
+    name: 'Created on ',
+    transform: dateFormatter.methods.formatDate,
+  },
+];
+
+export default Vue.extend({
+  components: {
+    GirderDetailList,
+    GirderMarkdown,
+  },
+  mixins: [sizeFormatter, usernameFormatter],
+  props: {
+    value: {
+      required: true,
+      type: Array,
+    },
+    infoKeys: {
+      type: Array,
+      default: () => DefaultInfoKeys,
+    },
+  },
+  data() {
+    return {
+      showUpsert: false,
+    };
+  },
+  inject: ['girderRest'],
+  asyncComputed: {
+    async details() {
+      if (this.datum && this.datum.created) {
+        return this.datum;
+      } if (this.datum && this.datum._id && this.datum._modelType) {
+        const { data } = await this.girderRest.get(`${this.datum._modelType}/${this.datum._id}`);
+        return data;
+      }
+      return null;
+    },
+  },
+  computed: {
+    title() {
+      return this.details
+        ? (this.details.name || this.formatUsername(this.details))
+        : `${this.value.length} Selection(s)`;
```

Comment:
maybe a specialized display for Object that is just `type:"root"` and `type:"'collections"` instead of the ` Selections(s)
![image](https://user-images.githubusercontent.com/61746913/113619603-2eeb5e00-9627-11eb-983d-261ac3ed0691.png)

![image](https://user-images.githubusercontent.com/61746913/113619797-6c4feb80-9627-11eb-864a-038f6bca350a.png)